Sources/VLCEditController.swift

```diff
@@ -9,15 +9,29 @@
  * Refer to the COPYING file of the official project for license.
  *****************************************************************************/
 
-class VLCEditController: NSObject {
+protocol VLCEditControllerDelegate: class {
+    func editController(editController: VLCEditController, cellforItemAt indexPath: IndexPath) -> MediaEditCell?
+}
+
+class VLCEditController: UIViewController {
     private var selectedCellIndexPaths = Set<IndexPath>()
-    private let collectionView: UICollectionView
     private let model: MediaLibraryBaseModel
+    weak var delegate: VLCEditControllerDelegate?
+
+    override func loadView() {
+        super.loadView()
```

Comment:
I think that the super call shouldn't be called here.

As the documentation stated: 

```You can override this method in order to create your views manually. If you choose to do so, assign the root view of your view hierarchy to the view property. The views you create should be unique instances and should not be shared with any other view controller object. Your custom implementation of this method should not call super.```